package.json

```diff
@@ -24,6 +24,7 @@
     "react-router-dom": "^5.2.0",
```

Comment:
just an idiomatic question, why using `^` here ?

Reply:
It means that this library requires *compatible version* of `react-router-dom` (means from v5.2.0 to <6.0.0)

Reply:
yeah `^` allow minor upgrades, and `~` only patches (the library is following semantic versioning of course).

I always use something that allow only patch level upgrades. I there any pros/cons about this in `javascript` world ?

PS : It is mainly a global reflexion, rather that related to this `PR`

Reply:
It's basically to tell `npm` which versions of the dependencies are allowed to be installed when doing a fresh `npm install`,  say you have `^5.2.0` on the dependencies, `npm` will install latest `5.x.x` version when doing a fresh `npm install`. Using `~` should be better since we are forced to bump minor version if our app is using a features that's only available on newer minor version (for clarity sake), but using `^` is fine most of the time.

Reply:
honestly, I'm have more `ruby` usage ... just curiosity about that ... neither strong opinion nor feedback 😛